openpype/modules/clockify/clockify_module.py

```diff
@@ -74,6 +62,8 @@ def tray_start(self):
         self.bool_workspace_set = self.clockapi.workspace_id is not None
         if self.bool_workspace_set is False:
             return
+        self.workspace_id = self.clockapi.workspace_id
```

Comment:
Attributes `workspace_id` and `user_id` are filled only if module is initialized in tray. This is by design not right. Ideally you should use the `clockapi` attribute when you need the values elsewhere.

If there is some reason, other than simplify access to them, you should add property methods that would return the value from `clockapi` instead. because I don't see any reason then that. Storing the value directly to module object can cause issue if something would change in clockify object.

```
@property
def workspace_id(self):
    return self.clockapi.workspace_id

@property
def user_id(self):
    return self.clockapi.user_id
```

Reply:
This is still an issue. Your storing values of clockapi object to clockify module object... That should not happen.